src/utils/DesignTokens/themes/dark.ts

```diff
@@ -354,6 +354,41 @@ export const darkTheme: DesignTokens = {
     sort: {
       'sort-icon-color': darkColors.white,
       'sort-icon-size': '14px'
+    },
+    typography: {
```

Comment:
This feels odd having the typography in the other place and here as well. Not sure if I would put this in that section tho since they are components and thats just design tokens. Just a thought.